src/igvfd/schemas/organoid.json

```diff
@@ -0,0 +1,123 @@
+{
+    "title": "Organoid",
+    "$id": "/profiles/organoid.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting an organoid sample.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/alternate_accessions"
+        },
+        {
+            "$ref": "mixins.json#/accessioned_status"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/submitter_comment"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
+        ],
+        "post_differentiation_time": {
+            "required": [
+                "post_differentiation_time_units"
+            ],
+            "comment": "Biosample with a specified post_differentiation_time requires post_differentiation_time_units specification."
+        },
+        "post_differentiation_time_units": {
+            "required": [
+                "post_differentiation_time"
+            ],
+            "comment": "Biosample with a specified post_differentiation_time_units requires post_differentiation_time specification."
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "post_differentiation_time": {
```

Comment:
Should this be replaced with "$ref"/in_vitro.json#/properties". if We think that all of in_vitro should be used in orgaoid, then yes.